tests/scripts/travis_setup.sh

```diff
@@ -1,13 +1,11 @@
 #!/bin/bash
 
-# Java 8, if needed.
-if [ $FEDORA_VERSION = "3.8.1" ]; then
-  sudo add-apt-repository -y ppa:webupd8team/java
-  sudo apt-get update
-  sudo apt-get install -y oracle-java8-installer oracle-java8-set-default
-  sudo update-java-alternatives -s java-8-oracle
-  export JAVA_HOME=/usr/lib/jvm/java-8-oracle
-fi
+# Java 8
```

Comment:
It certainly should. It did for me, in Travis. That having been said, we can certainly refine this to use a matrix formulation for Java version as we do for Fedora version.